tests/std/tests/Dev11_0836436_get_time/test.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#include <algorithm>

```

Comment:
CI format checks made me put this at the top. I guess it is just alphabetical order.

Reply:
Yes, our clang-format setup sorts headers.